src/rowtransforms.jl

```diff
@@ -0,0 +1,50 @@
+struct NormalizeRow{T, S} <: DataAugmentation.Transform
+        normstats::T
+        normcols::S
+end
+NormalizeRow(normstats::T, normcols::S) where {T, S} = NormalizeRow{T, S}(normstats, normcols)
+
+struct Categorify{T, S} <: DataAugmentation.Transform
+        catdict::T
+        categorycols::S
+end
+Categorify(catdict::T, categorycols::S) where {T, S} = Categorify{T, S}(catdict, categorycols)
+
+struct FillMissing{T, S} <: DataAugmentation.Transform
+        fmvals::T
+        fmcols::S
+end
+FillMissing(fmvals::T, fmcols::S) where {T, S} = FillMissing{T, S}(fmvals, fmcols)
+
+function DataAugmentation.apply(tfm::FillMissing, item::TabularItem; randstate=nothing)
+    x = [val for val in item.data]
+    for col in tfm.fmcols
+            idx = findfirst(col .== item.columns)
+            if ismissing(x[idx])
+                x[idx] = tfm.fmvals[col]
+            end
+    end
+    x = (; zip(item.columns, x)...)
+    TabularItem(x, item.columns)
+end
+
+function DataAugmentation.apply(tfm::NormalizeRow, item::TabularItem; randstate=nothing)
+    x = [val for val in item.data]
+    for col in tfm.normcols
+            idx = findfirst(col .== item.columns)
+            colmean, colstd = tfm.normstats[col]
+            x[idx] = (x[idx] - colmean)/colstd
+    end
+    x = (; zip(item.columns, x)...)
+    TabularItem(x, item.columns)
+end
+
+function DataAugmentation.apply(tfm::Categorify, item::TabularItem; randstate=nothing)
+    x = [val for val in item.data]
+    for col in tfm.categorycols
+        idx = findfirst(col .== item.columns)
+        x[idx] = ismissing(x[idx]) ? 1 : findfirst(skipmissing(x[idx] .== tfm.catdict[col])) + 1
```

Comment:
```suggestion
        x[idx] = ismissing(x[idx]) ? 1 : findfirst(x[idx] .== tfm.catdict[col]) + 1
```

No need for `skipmissing` here, right? `x[idx]` is a value and `tfm.catdict[col]` is a vector of categorical values (which doesn't contain `missing`). `findfirst` is just assigning an index based on which symbol in `tfm.catdict[col]` matches `x[idx]`.

Reply:
Initially I was thinking if someone creates `catdict` using `unique` or something, and if somehow `missing` is a part of this vector then an error could be thrown, but yeah it might just be better to remove it.

Reply:
Probably better to `map(v -> filter!(!ismissing, v), values(catdict))` when constructing the transform. We could throw a warning when that happens too.